src/main.rs

```diff
@@ -474,6 +472,10 @@ fn main() {
     let output_path = match output_types.len() {
         0 => return,
         1 => opt.output_path.as_deref(),
+        //TODO: Remove false positive clippy warning when
```

Comment:
Could you file an issue so we don't forget to do this?

Reply:
Yes I can make an issue and also added it the TODO for reference.
On it.

Reply:
#980